packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/types.ts

```diff
@@ -0,0 +1,26 @@
+import { type MaybeRef, type MaybeRefOrGetter } from "vue";
+import type { DataGridEntry } from "../../types";
+
+export type SortDirection = "asc" | "desc" | "none";
+
+/**
+ * -1: less than, 0: equal to, 1: greater than
+ */
+export type Compare<T> = (a: T, b: T) => -1 | 0 | 1;
+
+export type SortState<TEntry extends DataGridEntry> = {
+  column: keyof TEntry | undefined;
```

Comment:
```suggestion
  column?: keyof TEntry;
```

Isn't this the same?

Reply:
Okay I get it :) I am just wondering if this is weird to the user that he has to set undefined explicitly